src/components/facets.jsx

```diff
@@ -38,58 +38,64 @@ const parse = string => {
  * and generate a search string
  * @param {object} parsedSearch
  */
-const stringify = ({ facets = {}, ...rest }) => {
+export const stringify = (query, queryStringKey = 'facets') => {
```

Comment:
`queryStringKey = 'facets'` already listed in `Facets.defaultProps`

Reply:
This is for the helper function, whereas the default prop is for the component. Just in case we end up using the function on its own outside of the component